### ISSUE TITLE
LastTimestamp, update vs create

### ISSUE THREAD

**Issue by danielwertheim** [JasperFx/marten#695]
I understand that I could get the time stamp for created by looking at the "first" event and inspect that time stamp, but I think it would be nice to have this on the `StreamState`. Or is the recommendation to track this in a separate model if I have a need for it, e.g. for getting the streams that where created during a certain period?

**Comment by jeremydmiller** [comment]
Just an oversight. Shouldn't be that big of deal to add that to StreamState.

**Comment by jamesfarrer** [comment]
@jeremydmiller I am looking to get my first commit and this looks a good candidate. How were you envisaging doing this - add a timestamp column to mt_streams for created time or join with mt_events to get the earliest timestamp?

**Comment by jeremydmiller** [comment]
@jamesfarrer I think it'd be a lot easier to query on to just write it to the mt_streams table. Can you do this against the 2.0 branch? I *think* I need to finally get onto the better story for database schema change detection before we can roll this out and that's going to happen in the 2.0 branch.
